### PR TITLE
fix standard sql with !=

### PR DESCRIPTION
**Pull request by typhoonzero**

### CODE REVIEW

File: sql/lexer.go [hunk 1]
```diff
@@ -86,7 +86,7 @@ func (l *lexer) Lex(lval *sqlSymType) int {
 		return l.lexNumber(lval)
 	case r == '"':
 		return l.lexString(lval)
-	case strings.IndexRune("+-*/%<>=()[]{},;", r) >= 0:
+	case strings.IndexRune("+-*/%<>=()[]{},;!", r) >= 0:
```
Comment (Yancey0623) [inline sql/lexer.go:89]: Add some unit test?

File: sql/sql.y [hunk 2]
```diff
@@ -135,7 +135,7 @@
 %token <val> IDENT NUMBER STRING
 
 %left <val> AND OR
-%left <val> '>' '<' '=' GE LE
+%left <val> '>' '<' '=' '!' GE LE NE
```
Comment (weiguoz) [inline sql/sql.y:138]: Since `NE` is supported, do we need to support `EQ`?

Comment (typhoonzero) [reply sql/sql.y:138]: EQ is supported by line 269: `| expr '=' expr  { $$ = binary('=', $1, $2, $3) }`. We may add `EQ` when we need it?

**Review by weiguoz — APPROVED** [review]
LGTM

**Pull request merged**